src/main/java/io/strimzi/test/container/utils/KafkaVersionService.java

```diff
@@ -28,11 +23,7 @@
 public class KafkaVersionService {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(KafkaVersionService.class);
-
-    private static final Pattern STRIMZI_TEST_CONTAINER_IMAGE_WITHOUT_KAFKA_VERSION = Pattern.compile("^test-container:(\\d+\\.\\d+\\.\\d+|latest)-kafka-.*$");
-    private static final String KAFKA_VERSIONS_URL_JSON = "https://raw.githubusercontent.com/strimzi/test-container-images/main/kafka_versions.json";
-    private static final String STRIMZI_BASE_IMAGE = "quay.io/strimzi-test-container/test-container";
-    private static final String IMAGE_FORMAT = "%s:%s-kafka-%s";
+    private static final Properties PROPERTIES = System.getProperties();
```

Comment:
Any special reason why to store this instead of just calling `System.getProperties()` in the place where you use it?